### DISCUSSION TITLE
安裝homebrew遇到的問題

### DISCUSSION THREAD

**Discussion by liangyinglly** [1101-GS4719-C/Python-Community-GS4719-C#661] [Q&A]
Homebrew是簡化 macOS 安裝上可以更便利的工具
因為最近想安裝一些下載流程比較複雜的軟體
所以想說嘗試安裝homebrew看看
但是我遇到了這個問題 不知道有沒有人能來救救我 我該怎麼辦

<img width="568" alt="截圖 2022-06-10 上午11 03 53" src="https://user-images.githubusercontent.com/99886694/172982142-d90289d8-c5dc-4c9c-b536-c8dcc32848ab.png">

**Comment by treeann** [comment]
我也遇過這個問題求幫助

**Comment by gnsJhenJie** [comment]
有可能是網路連線不穩的問題, 也可以試試看下這個指令`git config --global http.postBuffer 1048576000`調整git的post buffer之後再重新安裝看看。

**Reply by gnsJhenJie** [reply]
然後建議可以用複製貼上的方式把錯誤訊息貼上來, 這樣找起問題會方便一些~
